extensions/railway/src/railway-projects.tsx

```diff
@@ -45,22 +45,21 @@ const ListProjects: React.FC = () => {
           actions={
             <ActionPanel>
               <ActionPanel.Section>
-                <Action.OpenInBrowser title="Project Settings" url={projectUrl(p.id, "settings")} />
-                <Action.OpenInBrowser title="Project Deployments" url={projectUrl(p.id, "deployments")} />
+                <Action.OpenInBrowser title="Project Architecture" url={projectUrl(p.id)} />
                 <Action.OpenInBrowser
-                  title="Latest Deployment"
-                  url={projectUrl(p.id, "deployments?open=true")}
-                  shortcut={{ modifiers: ["cmd"], key: "l" }}
+                  title="Project Settings"
+                  url={projectUrl(p.id, "settings")}
+                  shortcut={{ modifiers: ["cmd"], key: "s" }}
                 />
                 <Action.OpenInBrowser
-                  title="Project Variables"
-                  url={projectUrl(p.id, "variables")}
-                  shortcut={{ modifiers: ["cmd"], key: "v" }}
+                  title="Project Observability"
+                  url={projectUrl(p.id, "observability")}
+                  shortcut={{ modifiers: ["cmd"], key: "o" }}
                 />
                 <Action.OpenInBrowser
-                  title="Project Metrics"
-                  url={projectUrl(p.id, "metrics")}
-                  shortcut={{ modifiers: ["cmd"], key: "m" }}
+                  title="Project Logs"
+                  url={projectUrl(p.id, "logs")}
+                  shortcut={{ modifiers: ["cmd"], key: "l" }}
```

Comment:
Should we mention the new keyboard shortcuts in `CHANGELOG`? It's a change for those who use shortcuts a lot.

Reply:
Yes, absolutely. Added it ✅. Just fyi, all the links and shortcuts I changed were 404s, so I thought changing them would do no harm.

Reply:
Yup that's fine. I don't have any active projects in Railway rn so I can't test it but the video makes it seems like all's well 👍.